keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -401,6 +725,7 @@ friction.
 
 ## Alternatives
 
+### Localhost profiles
 The localhost feature currently depends on an alpha Kubelet flag. We could therefore label the
```

Comment:
would the alpha `seccomp-profile-root` kubelet flag no longer be considered alpha?

Reply:
IMO we should deprecate this flag, and use the same approach as various other local plugins - configurable root directory, with a fixed subdirectory. See https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/defaults.go and https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_getters.go#L88

Reply:
i would like this clarified as well.

the kubelet uses this to build a path like `localhost/<some-dir>/<name>` when passing the seccompfile path down to sandbox creation flows.

Reply:
Removing unnecessary degrees of freedom is good, so if stemming off of the kubelet `--root-dir` seems reasonable to sig-node, I'm all for deprecating the current flag and dropping it after a release.

Reply:
Made the change to deprecate on v1.19 and remove on v1.23 - the same version that will drop support for annotations.

Reply:
^ @derekwaynecarr to ack